tests/e2e/test_sync_app_data.py

```diff
@@ -1,80 +1,50 @@
 import os
-import shutil
 import unittest
-from pathlib import Path
+import time
+
 from unittest import IsolatedAsyncioTestCase
 
 from dotenv import load_dotenv
-from dune_client.file.interface import FileIO
+from dune_client.client import DuneClient
 
 from src.fetch.dune import DuneFetcher
-from src.models.block_range import BlockRange
-from src.sync.app_data import AppDataHandler, SYNC_TABLE
+from src.fetch.orderbook import OrderbookFetcher
+from src.sync import sync_app_data
 from src.sync.config import AppDataSyncConfig
 
 
 class TestSyncAppData(IsolatedAsyncioTestCase):
     def setUp(self) -> None:
         load_dotenv()
-        self.dune = DuneFetcher(os.environ["DUNE_API_KEY"])
-        self.config = AppDataSyncConfig(
-            volume_path=Path(os.environ["VOLUME_PATH"]),
-            missing_files_name="missing_app_hashes.json",
-            max_retries=2,
-            give_up_threshold=3,
+        self.dune = DuneClient(os.environ["DUNE_API_KEY"])
+        self.fetcher = DuneFetcher(self.dune)
```

Comment:
`DuneFetcher` does not seem to be used anywhere. Would it be safe to remove that class?